src/common/imagwebp.cpp

```diff
@@ -0,0 +1,346 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/common/imagwebp.cpp
+// Purpose:     wxImage WebP handler
+// Author:      Hermann Höhne
+// Copyright:   (c) Hermann Höhne
+// Licence:     wxWidgets licence
+/////////////////////////////////////////////////////////////////////////////
+
+// based on code by Sylvain Bougnoux, Khral Steelforge and Markus Juergens
+// see https://forums.wxwidgets.org/viewtopic.php?t=39212
+
+// For compilers that support precompilation, includes "wx.h".
+#include "wx/wxprec.h"
+
+#if wxUSE_IMAGE && wxUSE_LIBWEBP
+
+#include "wx/imagwebp.h"
+#include "webp/demux.h"
+#include "webp/decode.h"
+#include "webp/encode.h"
+#include <vector>
+
+#ifndef WX_PRECOMP
```

Comment:
Very minor, but I'd put this before all the other headers for consistency with the other files.